src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -181,15 +184,21 @@ private void handleExit() {
      * @see seedu.address.logic.Logic#execute(String)
      */
     private CommandResult executeCommand(String commandText) throws CommandException, ParseException {
-        if (commandText.equals("list")) { //TODO: Remove this in v1.3
-            showPersonList();
-        }
 
         try {
+
             CommandResult commandResult = logic.execute(commandText);
             logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
 
+            if (commandText.equals("list")) {
+                prevIsTagList = false;
+            }
```

Comment:
This issue is meant to remove the bad code `commandText.equals("list")`.
Suggestion is to implement `commandResult.isPersonList()` or its equivalent alternatives.